src/styles/catalog.scss

```diff
@@ -0,0 +1,95 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, 200px);
+  margin-top: 50px;
+  margin-bottom: 50px;
+  justify-content: center;
+  gap: 50px;
+
+  /* .catalog__card */
+
+  &__card {
+    width: 200px;
+    padding: 20px;
+    border: 1px solid #F3F3F3;
+    border-radius: 3px;
+    & img{
+      max-width: 160px;
+      margin-bottom: 20px;
+    }
+    h4{
+      margin-top: 5px;
+      color: #616070;
```

Comment:
The color variables can be defined in separate file and you can use them here and in other places.